mesonbuild/backend/ninjabackend.py

```diff
@@ -2263,6 +2263,10 @@ def generate_swift_target(self, target) -> None:
                 header_imports += swiftc.get_header_import_args(absh)
             else:
                 raise InvalidArguments(f'Swift target {target.get_basename()} contains a non-swift source file.')
+        if header_imports != []:
+            # Swift writes a PCH file for bridging headers. Without this flag, it puts it somewhere in /tmp.
+            # '.' is already the target private dir.
+            header_imports += swiftc.get_pch_output_dir_args('.')
```

Comment:
Looking at this again, I don't know if we actually want this. Might be better to leave these in the temp directory where they will be automatically cleaned up eventually, rather than the build directory where that is not the case until it is deleted. And just hope this is solved in a less stupid way by Swift in the future (such as not writing a PCH file to disk in the first place)

Reply:
Yeah. Away it goes.